src/python/zigopt/utils/create_produser.py

```diff
@@ -90,63 +89,3 @@ def execute_query(conn, cursor, query_string, query_args=None, error_allow_list=
           return False
     raise
   return True
-
-
-def parse_args():
-  parser = argparse.ArgumentParser(
-    description="Create produser",
-  )
-
-  parser.add_argument(
-    "--database",
-    default="basedb",
-  )
-
-  parser.add_argument(
-    "--host",
-    default=None,
-  )
-
-  parser.add_argument(
-    "--port",
-    default=None,
-  )
-
-  parser.add_argument(
-    "--superuser",
-    default="postgres",
-    help="The superuser used for sigopt-server's database service",
-  )
-
-  parser.add_argument(
-    "--superuser_password",
-    default=None,
-    help="The superuser's password",
-  )
-
-  parser.add_argument(
-    "--produser",
-    default="produser",
-    help="The name of the new produser to be created",
-  )
-
-  parser.add_argument(
-    "--produser_password",
-    default="produser_password_for_development",
-    help="The produser's password",
-  )
-
-  return parser.parse_args()
-
-
-if __name__ == "__main__":
```

Comment:
This change seems unrelated and or odd?

Reply:
This script was broken because `query` was a missing required argument for `create_produser`. This script is unused anyway